### PR TITLE
Make users know about telemetry via `startupWarnings`

### PR DESCRIPTION
**Pull request by rumyantseva**
Tests for https://github.com/FerretDB/FerretDB/pull/1336.
Issue: https://github.com/FerretDB/FerretDB/issues/1328.

### CODE REVIEW

File: tests/diff/startup_warnings_test.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+func TestStartupWarnigns(t *testing.T) {
+	t.Parallel()
+
+	// TODO How to enable and disable telemetry when FerretDB is running?
```
Comment (rumyantseva) [inline tests/diff/startup_warnings_test.go:28]: @AlekSi, and now I need some help understanding how to write dance diff tests for telemetry, as there is no way to set telemetry as true/false/undecided through db commands. So, what's the way to ensure that the telemetry state is set as we expect it?

Comment (AlekSi) [reply tests/diff/startup_warnings_test.go:28]: The current way is "wait for FerretDB/FerretDB#1323", I guess :)

**Review by AlekSi — COMMENTED** [review]
Please don't update mongo-go-driver in that PR

**Comment by rumyantseva** [comment]
@noisersup I suggest closing this PR for now as it focuses on `undecided`, and we currently don't have a decision about a good approach to test it.

**Pull request closed without merging**